csrankings-j.csv

```diff
@@ -1005,6 +1005,7 @@ Jianjun Chen 0005,Tsinghua University,https://jianjunchen.com,7ryl0L8AAAAJ
 Jianjun Hu,University of South Carolina,http://www.cse.sc.edu/~jianjunh,_iD3nyMAAAAJ
 Jianjun Huang 0001,Renmin University of China,https://hjjandy.github.io,ip0G7pgAAAAJ
 Jianke Zhu,Zhejiang University,http://mypage.zju.edu.cn/jkzhu,SC-WmzwAAAAJ
+Jianlei Yang 0001,Beihang University,https://dblp.org/pid/99/9547-1.html,HLEelmYAAAAJ

```

Comment:
DBLP entries don't qualify as home pages. Please re-open this PR with a non-anonymous Github alias (or add contact info to your avatar).